K3s notes (#2526)
mendersoftware/mender-docs: 07.Server-installation/04.Production-installation-with-kubernetes/02.Storage/docs.md

!! recommended for production: AWS S3, Cloudflare R2,
!! Google Cloud Storage,and Azure Blob Storage.

> !! Important: The object storage provider have to be resolvable both

Reviewer: ```suggestion
!! Important: The object storage provider has to be resolvable both
```

!! internally and externally. If you are using a private IP address,

> !! you have to make sure that the DNS resolution is working correctly.

Reviewer: ```suggestion
!! you must ensure DNS resolution is working correctly.
```

!! For k3s, you could add this entry to the CoreDNS ConfigMap:
!! ```yaml
!!     mender.example.com:53 {
!!      errors
!!      cache 30
!!      hosts {
!!        <ip address of your machine> mender.example.com
!!      }
!!    }
!! ```

> !! But please note that the TLS certificate for the host `mender.example.com`
> !! has to be valid and trusted by the clients.

Reviewer: ```suggestion
!! Note, the TLS certificate for the host `mender.example.com`
!! must be valid and trusted by clients.
```

Reviewer: It would be going above and beyond, but if you have reference for exactly how to obtain and install the certificate in the clients it would be a great place to link to it.

Reviewer 2: @nickanderson yes this is definitely something we should address.

Author: https://northerntech.atlassian.net/browse/MC-7739 here the issues
